packages/react-core/src/components/Timestamp/Timestamp.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Timestamp/timestamp';
+import { css } from '@patternfly/react-styles';
+import { Tooltip } from '../Tooltip';
+
+export enum TimestampFormat {
+  full = 'full',
+  long = 'long',
+  medium = 'medium',
+  short = 'short'
+}
+
+export enum TimestampTooltipVariant {
+  default = 'default',
+  custom = 'custom'
+}
+
+export interface TimestampTooltip {
+  /** The variant of the tooltip. The "default" variant displays the date passed into
+   * the timestamp's date prop as a UTC string.
+   */
+  variant: 'default' | 'custom';
+  /** Content displayed in the tooltip when using the "custom" variant. */
+  content?: React.ReactNode;
+  /** Additional props passed to the tooltip. */
+  tooltipProps?: any;
+  /** A custom suffix to apply to a "default" variant tooltip. */
+  suffix?: string;
+}
+
+export interface TimestampProps extends React.HTMLProps<HTMLSpanElement> {
+  /** Displays custom content inside the timestamp, such as a relative time. This prop will
+   * override the default content that is displayed. Custom content will not be affected by
+   * any props that format or add to displayed content.
+   */
+  children?: React.ReactNode;
+  /** Class to add to the outer span. */
+  className?: string;
+  /** Applies custom formatting to the displayed date and/or time. Passing this prop in will
+   * override the dateFormat and timeFormat props.
+   */
+  customFormat?: { [key: string]: string };
+  /** Determines the default content that is displayed inside the timestamp. If a dateTime prop is
```

Comment:
It looks like you need to update the prop description here

Reply:
Context for this description is I wanted to convey that consumers could pass in their own dateTime prop to the component to override what the component itself constructs internally. Re-reading it it can be a bit confusing since there's no dateTime prop declared, and that could possibly be something better suited for a11y or design docs. How does the following sound instead?

"Determines the default content that is displayed inside the timestamp and the value of the inner time element's datetime attribute"

Reply:
I think that is better.